tests/ui/abi/unsupported.rs

```diff
@@ -20,39 +20,120 @@
     abi_msp430_interrupt,
     abi_avr_interrupt,
     abi_x86_interrupt,
-    abi_riscv_interrupt
+    abi_riscv_interrupt,
+    abi_c_cmse_nonsecure_call
 )]
 #[lang = "sized"]
 trait Sized {}
 
+#[lang = "copy"]
+trait Copy {}
+
 extern "ptx-kernel" fn ptx() {}
 //~^ ERROR is not a supported ABI
+fn ptx_ptr(f: extern "ptx-kernel" fn()) {
+    //~^ ERROR is not a supported ABI
+    f()
+}
+extern "ptx-kernel" {}
+//~^ ERROR is not a supported ABI
```

Comment:
We expanded these tests to also cover `extern` blocks and function pointers, however this now got very repetitive. Maybe there is a way to reduce duplication?